src/main/java/hudson/plugins/scm/koji/KojiChangeLogSet.java

```diff
@@ -83,4 +81,60 @@ public User getAuthor() {
 
     }
 
+    private static List<Hyperlink> getListFromString(String string) {
+        List<Hyperlink> hyperlinks = new ArrayList<>();
+        hyperlinks.add(new Hyperlink(string));
+        return hyperlinks;
```

Comment:
this could be one-liner `return Collections.singletonList(new Hyperlink(string));` You could even call this on top, when creating list https://github.com/judovana/jenkins-scm-koji-plugin/pull/19/files#diff-4d3d97cc968413173da7115e5a33ce3eR25